### DISCUSSION TITLE
What is the nocodb equivalent to nuBuilder lookup?

### DISCUSSION THREAD

**Discussion by m2acgi** [nocodb/nocodb#2691] [Features]
Click the paly button in the upper right corner of the gif to play.

![2022-07-16_11-29-02](https://user-images.githubusercontent.com/34497122/179337498-31b40d67-ea66-40b5-bd1b-902d7e5c475a.gif)

**Comment by dstala** [comment]
@m2acgi, please refer [here](https://docs.nocodb.com/setup-and-usages/lookup)

**Reply by m2acgi** [reply]
I'll try it later. Thanks.
